woocommerce/woocommerce-ios
# [Hazmat Shipping] Introduce Hazmat section with Feature Flag

Closes #10551 and partially fix #10583

Why
==========
Some companies like USPS are starting to require that merchants declare when a hazardous material is inside shipped packages. As the mobile app contains the shipping label creation feature, we need to comply with this requirement in order to avoid issues between our merchants and package shipping companies. 

This PR introduces the first step of the UI section responsible to allow the merchant to declare hazmat shipping.

How
==========
Introduce a UI section that's not connected to no models or network operation yet, just the UI section itself with the visibility bound by a respective feature flag. The instructions area of the section doesn't contain the expected clickable links yet, and the `Select a category` button is still pending. All this will become available in the following PRs for better breakdown of reviews.

Screen Capture
==========
https://github.com/woocommerce/woocommerce-ios/assets/5920403/75bc7928-922a-4d44-9b67-66b70822c08f

How to Test
==========
1. Open the order details of an order eligible for Shipping label creation
2. Start the shipping label creation flow until you reach the Package details section
3. Inside the package details view, make sure the Contains hazardous materials toggle is visible and interactable
4. Make sure the toggle expands and collapses a basic view that will serve as the main control options for the HAZMAT declaration

Update release notes:
